### DISCUSSION TITLE
These tips will tell you that the specific internship is scam or not

### DISCUSSION THREAD

**Discussion by ashwinexe** [EddieHubCommunity/support#1854] [Q&A] [answered]
Hi. as a fresher its super exciting to get an internship. At the same time this is the most vulnerable phase to get caught in scam internships (Internships that don't pay you, consumes you time and may even steal your project without giving credits back)

It is highly recommended to research about the company that is offering the internship.
Here are some of the points to look for while researching:

Do add your points which will help others stuck in this phase

**Comment by ashwinexe** [comment, marked as answer]
Here are some things to look for (from my personal experience with a scam internship)
- Is there a website, if yes is it functional? (look for contact-us page and working links)
- Is the data proposed on website valid? (a simple google search can reveal that)
- Is the work culture suspicious in the company (Does it feel like a corporate/professional environment or is not managed well)
- Will you paid in advanced? or is their any confirmation that you will be paid? 
- Are their any compaints/negative reviews against the company? (again, google can help)

**Reply by ghost** [reply]
@ashwinexe did you forgot to mark this as the answer?

**Reply by ashwinexe** [reply]
I Was waiting maybe more people could enter their thoughts

**Reply by ghost** [reply]
> I Was waiting maybe more people could enter their thoughts

You can still mark it as answered people will feel more faithful in your message, folks will keep adding their cents even after, don't worry, plus I'll link it to #help so that folks can get exposure to this!

**Reply by vijay-jaisankar** [reply]
Also, a heap of positive reviews which seem engineered could be a thing to consider. wdys @ashwinexe

**Reply by ashwinexe** [reply]
> Also, a heap of positive reviews which seem engineered could be a thing to consider. wdys @ashwinexe 

I've never seen that instance in this field of industry. Does that happen a lot?
